fontir/src/ir.rs

```diff
@@ -426,13 +426,18 @@ impl GlobalMetricsBuilder {
     ) {
         let units_per_em = units_per_em as f64;
 
-        let mut set_if_absent = |metric, value| self.set_if_absent(metric, pos.clone(), value);
+        // a macro instead of a closure to avoid borrowck
```

Comment:
This smells, I didn't spot it at a glance, why can't this be normal code?

Reply:
Personally, I think calling `self.set_if_absent(metric, pos.clone(), value);` directly is fine.

- set_if_absent can also take an `Into<Cow<NormalizedLocation>>` if you want to avoid the explicit clone

Reply:
The whole reason this exists afaict is because always calling `self.set` ends up just... being a bunch of extra lines? And the closure doesn't work with this patch because you can't call `self.get` because the closure has an `&mut` ref to self.